contract/src/main/java/com/ritense/valtimo/contract/json/patch/JsonPatchBuilder.java

```diff
@@ -82,8 +84,30 @@ public JsonPatchBuilder copy(JsonPointer from, JsonPointer to) {
         return this;
     }
 
+    /** Adds a JsonNode value to a json at the specified location. */
+    public JsonPatchBuilder addJsonNodeValue(JsonNode destination, JsonPointer path, JsonNode value) {
```

Comment:
Not entirely sure if this logic is sound, at the very least it doesn't read nicely. Can you add some comments to make this easier to read?

Reply:
well i don't really know about that stuff. i dont really know where that actually came from